addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -350,7 +350,8 @@
                 <field name="workcenter_id"/>
                 <field name="product_id"/>
                 <field name="qty_production"/>
-                <field name="product_uom_id"/>
+                <field name="product_uom_id" readonly="0"/>
```

Comment:
You could use force_save instead of readonly=0 because I think, we don't want to let the user modify it